presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -364,44 +364,44 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .aggregate(CovarianceAggregation.class)
                 .aggregate(RegressionAggregation.class)
                 .aggregate(CorrelationAggregation.class)
-                .scalar(SequenceFunction.class)
```

Comment:
I think it is good change:
 1) If you wan't this name to be the same than we have to restore code for recognizing which version do we wan't. This causes many tricky scenarios and lowers the possibility of finding user error while parsing this.
 2) This are two semantically different classes types. We don't ever use same class for both of this things. Code recognizing that should check for all different behaviors like defining some parts of both classes types. It's tricky to make it bullet proof. Of course we don't need it bullet proof. However by separating that (so that it matches the semantic), we have that for free - user defines what does he try to do and if he do something different checks will force him to fix this.
 3) Philosophically: I think one function doing two different things is against OOP rules.
 4) By the definition of the word, you may call every refactoring `unnecessary`, but I don't think there is value in minimizing refactors as such.
 5) To be consistent I would rather continue this changes to the rest of methods (I'm about to start refactoring + extending annotation support for aggregate functions as a followup of https://github.com/prestodb/presto/pull/5381